src/fosslight_source/run_scancode.py

```diff
@@ -47,14 +48,36 @@ def main():
                 output_file = arg
             elif opt == "-m":
                 print_matched_text = True
+            elif opt == "-x":
```

Comment:
How to select Scanner as a parameter needs discussion.

Reply:
Let's leave it as is and change once decisions on parameter for fosslight_scanner are cleared.